test.js

```diff
@@ -33,7 +33,6 @@ describe('capture-exit', function() {
   describe('.captureExit', function() {
     afterEach(function() {
       // always restore, in case we have bugs in our code while developing
-      process.exit = originalExit;
```

Comment:
as mentioned in the commit message: the outer `afterEach()` block is doing the same thing already. I don't see a reason to reset this twice.